### PR TITLE
Query and pass down index level list of location.lvl1 facet values. U…

### PR DESCRIPTION
**Pull request by mcstover**
…se this list to persist location select options when other facets (attributes, tags, gender, etc)  are used for filtering.

### CODE REVIEW

**Comment by mcstover** [comment]
Initial setup is here and seems to work well. This does not yet set the options with 0 loans to "disabled".

**Comment by mcstover** [comment]
FYI @emuvente @BoulderBrains @foochris

**Comment by BoulderBrains** [comment]
👀

File: src/plugins/algolia-init-mixin.js [hunk 1]
```diff
@@ -184,5 +189,33 @@ export default {
...
+
+		// initialize utility instance of default index
+		this.defaultIndexInstance = this.searchClient.initIndex(this.algoliaDefaultIndex);
+
+		// set global data set for Lvl1 Locations
+		this.setAllLvl1Locations();
 	},
+	methods: {
+		setAllLvl1Locations() {
+			this.defaultIndexInstance.searchForFacetValues({
```
Comment (ksekhri) [inline src/plugins/algolia-init-mixin.js]: Cool solution :+1:

File: src/pages/Lend/Filter/FilterSections/LocationMulti/LocationMultiRefinements.vue [hunk 2]
```diff
@@ -47,6 +47,7 @@
 				:checked="isCountryChecked(item.isRefined, item.region)"
 				:label="`${item.label} (${item.count})`"
 				@checkbox-input="handleCountryCheckboxInput(item)"
+				:disabled="item.count === 0"
```
Comment (ksekhri) [inline src/pages/Lend/Filter/FilterSections/LocationMulti/LocationMultiRefinements.vue]: Nice

**Pull request merged**